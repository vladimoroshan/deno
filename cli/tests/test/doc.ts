/**
 * ```
 * import { example } from "./doc.ts";
 *
 * console.assert(example() == 42);
 * ```
 */
export function example(): string {
  return "example";
}
