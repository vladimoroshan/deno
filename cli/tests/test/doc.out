Check [WILDCARD]/doc.ts$2-7
error: TS2367 [ERROR]: This condition will always return 'false' since the types 'string' and 'number' have no overlap.
console.assert(example() == 42);
               ~~~~~~~~~~~~~~~
    at [WILDCARD]/doc.ts$2-7.ts:3:16
